packages/v4/src/content/design-guidelines/components/tree-view/tree-view.md

```diff
@@ -3,5 +3,92 @@ id: Tree view
 section: components
 ---
 
-We’re working on some guidelines for this component.
-If you have any questions in the meantime, feel free to [get in touch with us.](/get-in-touch)
+A **tree view** is a structure that displays data in a hierarchical view. It can be used in a [dropdown](/components/dropdown), [drawer](/components/drawer), [primary-detail](/demos/primary-detail), [modal](/components/modal), or [wizard](/components/wizard).
+
+A default tree view provides a tree structure with items and an arrow to expand or collapse child items. Every tree view can be extended with these [optional elements](#tree-view-capabilities).
+
+## Tree view elements
+
+A default tree view consists of 5 elements:
+
+<img src="./img/tree-view-elements.png" width="332" />
+
+1. **Expand/collapse button:** A button that allows you to expand or collapse child nodes 
+2. **Parent node:** A node that contains other items (for example, a folder that contains multiple documents)
+3. **Child node:** A node within the parent node (for example, a document within a folder)
+4. **Leaf node:** The last item in a tree, which doesn't have any child nodes
+5. **Truncation indicator:** A control that automatically shortens long names 
+
+## Usage
+
+A tree view can be used for: 
+1. Changing the content on a page based on the selected item. 
+2. Selecting items or applying filters. 
+
+<img src="./img/Default-tree-view.png" width="1004" />
+
+Use a tree view when:   
+* Selecting is the main use-case. 
+* You apply filters that need to be structured in a clear hierarchy. 
+* The data is structured into levels.
+* You need to display hierarchies that have more than 2 levels.
+
+Don’t use a tree view when: 
+* The items are not structured hierarchically. Use the [data list](/components/data-list) instead. 
```

Comment:
or a menu list too? or simple list? Or would you say that's not the same use case as tree view. I just feel like data lists aren't primarily used as selection tools

Reply:
You're right. I'll change it. Do you mean this menu list? https://www.patternfly.org/v4/components/menu
I found a simple list.

Reply:
Yes although maybe a menu (which actually is just used in drop downs or select lists) doesn't make sense for the tree view use case-  so nevermind, simple list should be enough, thank you!!